app/controllers/alerts_list_controller.rb

```diff
@@ -0,0 +1,40 @@
+class AlertsListController < ApplicationController
+  extend ActiveSupport::Concern
+
+  before_action :check_privileges
+  before_action :session_data
+  after_action :cleanup_action
+  after_action :set_session_data
+
+  def show
+    if params[:id].nil?
+      @breadcrumbs.clear
+    end
+  end
+
+  def index
+    redirect_to :action => 'show'
+  end
+
+  def class_icons
+    res = {}
+    [
+      'ManageIQ::Providers::Kubernetes::ContainerManager::ContainerNode',
+      'ManageIQ::Providers::Openshift::ContainerManager',
```

Comment:
possibly missing `ManageIQ::Providers::Hawkular::DatawarehouseManager` ? (you do have it in the test response)